core/PRP/Geometry/plGBufferGroup.cpp

```diff
@@ -547,6 +560,113 @@ void plGBufferGroup::addIndices(const std::vector<unsigned short>& indices)
         fIdxBuffStorage[idx][i] = indices[i];
 }
 
+void plGBufferGroup::IPackIcicle(const plGeometrySpan* geoSpan, plIcicle* ice)
+{
+    size_t i;
+    for (i = 0; i < fIdxBuffStorage.size(); ++i)
+        if (geoSpan->getNumIndices() + fIdxBuffCounts[i] < kMaxNumIndicesPerBuffer)
+            break;
```

Comment:
Please use braces around multiline loop bodies, even if they are not syntactically necessary.